jobs/pull-kubernetes-federation-e2e-gce-canary.env

```diff
@@ -0,0 +1,27 @@
+# Panic if anything mutates a shared informer cache
+ENABLE_CACHE_MUTATION_DETECTOR=true
+
+FEDERATION=true
+USE_KUBEFED=true
+PROJECT=k8s-jkns-pr-bldr-e2e-gce-fdrtn
```

Comment:
If you choose a different GCE project, we might need to add the deploy job too.
Also the test jobs are made not run on all PR's and will be triggered only when they are mentioned. I think it should be ok right? its like maybe few more PR's being tested